lib/daru/io/importers/redis.rb

```diff
@@ -0,0 +1,156 @@
+        #
+        # @example Importing with Redis instance without specifying keys
+        #   # Say, the Redis connection has this setup
+        #   # Key "name"   => ["Tyrion", "Jamie", "Cersei", "Joffrey"]
+        #   # Key "age"    => [32, 37, 37, 19]
+        #   # Key "living" => [true, true, true, false]
+        #
+        #   # Say `connection` is a Redis instance
+        #   df         = Daru::DataFrame.from_redis(connection)
+        #
+        #   df
+        #
+        #   #=> <Daru::DataFrame(4x3)>
+        #   #=>           name     age  living
+        #   #=>      0  Tyrion      32    true
+        #   #=>      1   Jamie      37    true
+        #   #=>      2  Cersei      37    true
+        #   #=>      3 Joffrey      19   false
+        #
+        # @example Importing with Redis instance by specifying keys
+        #   # Say, the Redis connection has this setup
+        #   # Key "name"   => ["Tyrion", "Jamie", "Cersei", "Joffrey"]
+        #   # Key "age"    => [32, 37, 37, 19]
+        #   # Key "living" => [true, true, true, false]
+        #
+        #   # Say `connection` is a Redis instance
+        #   df         = Daru::DataFrame.from_redis(connection, "name", "age")
+        #
+        #   df
+        #
+        #   #=> <Daru::DataFrame(4x2)>
+        #   #=>           name     age
+        #   #=>      0  Tyrion      32
+        #   #=>      1   Jamie      37
+        #   #=>      2  Cersei      37
+        #   #=>      3 Joffrey      19
+        #
+        # @example Querying for matching keys with count
+        #   # Say, the Redis connection has this setup
+        #   # Key "timestamp:100620171225" => { "name" => "Joffrey", "age" => 19 }.to_json
+        #   # Key "timestamp:090620171222" => { "name" => "Cersei",  "age" => 37 }.to_json
+        #   # Key "timestamp:090620171218" => { "name" => "Jamie",   "age" => 37 }.to_json
+        #   # Key "timestamp:090620171216" => { "name" => "Tyrion",  "age" => 32 }.to_json
+        #
+        #   # Say `connection` is a Redis instance
+        #   df = Daru::DataFrame.from_redis(connection, match: "timestamp:09*", count: 3)
+        #
+        #   df
+        #
+        #   #=> <Daru::DataFrame(3x2)>
+        #   #=>                               name     age
+        #   #=>      timestamp:090620171222  Cersei      37
+        #   #=>      timestamp:090620171218   Jamie      37
+        #   #=>      timestamp:090620171216  Tyrion      32
+        #
+        def initialize(connection={}, *keys, match: nil, count: nil)
+          super(binding)
+          @client = get_client(connection)
+          @keys   = choose_keys(@keys).map(&:to_sym)
+        end
+
+        def call
+          vals = @keys.map { |key| ::JSON.parse(@client.get(key), symbolize_names: true) }
+          Importer.guess_parse @keys, vals
+        end
+
+        private
+
+        def choose_keys(keys)
+          if keys.count.zero?
+            @client.scan(0, match: @match, count: @count).last
```

Comment:
Are you aware of the fact that response is paginated?..

https://redis.io/commands/scan
> SCAN may return a maximum number of elements in the order of a few tens of elements when iterating a large collection